File: typescript/two-fer/two-fer.ts
/**
 * 'Two-fer' is short for 'two for one'.
 * Given an optional name this funstion returns the message:
 * One for (name), one for me.
 */

export function twoFer(name = 'you'): string {
  return `One for ${name}, one for me.`;
}

File: python/two-fer/two_fer.py
"""
Functions for determining what to say when giving out an extra cookie from the two-fer one sale.
"""
def two_fer(name="you"):
    """Returns the phrase, 'one for you, one for me' customized with the given name.

    :param name: string - the name to use in the customized phrase, defaults to 'you' when not provided.
    :return: string - the customized phrase.
    """
    return f"One for {name}, one for me."
